nba_core/endpoints/endpoint_functions.py

```diff
@@ -0,0 +1,133 @@
+from flask import redirect, render_template, request, flash, session, g
```

Comment:
I would break this into a file/module for every function -- thats is opinion/preference. They would have names like `home|index.py`, `login.py`, `logout.py`, etc. If you want to keep this thats ok too, I would just change the module structure to be `nba_core/endpoints.py`. Same idea but with a single file/module instead of a directory since you would have only one file. Either way 👍